dist.ini

```diff
@@ -0,0 +1,69 @@
+name    = Math-Util-CalculatedValue
+author  = binary.com <BINARY@cpan.org>
+license = Perl_5
```

Comment:
You have changed license from Apache to Perl5 :) I think you should mention that in changelog